docs/test/action-groups.md

```diff
@@ -180,6 +180,34 @@ MFTF resolves `{{myCustomEntity.field1}}` the same as it would in a `selector` o
 </actionGroup>
 ```
 
+## Return a value
+
+Action groups can return a value using a `return` tag.
+
+```xml
+<actionGroup name="GetOrderIdActionGroup">
+    <seeElement selector="{{CheckoutSuccessMainSection.orderLink}}"  stepKey="assertOrderLink"/>
+    <grabTextFrom selector="{{CheckoutSuccessMainSection.orderLink}}" stepKey="orderId"/>
+    <return value="{$orderId}" stepKey="returnOrderId"/>
+</actionGroup>
+```
+
+The value returned can be accessed in later steps using action group step key `{$getOrderId}`.
+```xml
+<actionGroup ref="GetOrderIdActionGroup" stepKey="getOrderId"/>
+<!--Filter the Order using Order ID -->
+<actionGroup ref="FilterOrderGridByIdActionGroup" stepKey="filterOrderGridById">
+    <argument name="orderId" value="{$getOrderId}"/>
+</actionGroup>
+```
+### Convention to return a value 
```

Comment:
@jilu1 did you check this documentation. Does this answer the question regarding multiple returns and merging action groups?

Reply:
Yes. It looks good.